package.json

```diff
@@ -49,8 +48,8 @@
   "homepage": "https://github.com/fortinet/fortigate-autoscale-azure#readme",
   "dependencies": {
     "@azure/functions": "^1.2.3",
+    "@fortinet/fortigate-autoscale": "https://github.com/fortinet/autoscale-core/releases/download/3.2.0/fortinet-fortigate-autoscale-3.2.0.tgz",
```

Comment:
Use npm link during dev and have a script that updates the version number when you publish a new version?

Reply:
at some point we should stop goofing around with this and actually set up the npm registry at github?

Reply:
there is one and will change it to update this line in the next change.